jobQueue.py

```diff
@@ -79,7 +79,7 @@ def add(self, job):
             self.log.info("add|JobQueue is full")
             return -1
         self.log.debug("add|Gotten next ID: " + str(job.id))
-        self.log.info("add|Unassigning job %s" % str(job.id))
+        self.log.info("add|Unassigning job ID: %d" % (job.id))
```

Comment:
Didn't the previous statement still show the job ID? `self.log.info("add|Unassigning job %s" % str(job.id))`